sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -160,6 +161,7 @@ AZ_NODISCARD az_result az_iot_hub_client_sas_signature_get(
 AZ_NODISCARD az_result az_iot_hub_client_sas_password_get(
     az_iot_hub_client const* client,
     az_span base64_hmac_sha256_signature,
+    uint32_t token_expiration_epoch_time,

```

Comment:
>token_expiration_epoch_time [](start = 13, length = 27)

should this value of expiry be same as passed to az_iot_hub_client_sas_signature_get?

Reply:
yes, absolutely. I'll add a clarification in the function doc.